src/components/TabletApp/RightSide/AdvancedBook/AdvancedBook.tsx

```diff
@@ -225,7 +230,15 @@ const AdvancedBook = () => {
                                 sx={{ maxWidth: 140 }}
                                 type="submit"
                                 variant="contained"
-                                color="success"
+                                color={
+                                    availability ===
+                                    CONSTANTS.MEETING_WILL_FOLLOW
+                                        ? "warning"
+                                        : availability ===
+                                          CONSTANTS.MEETING_IN_PROGRESS
+                                        ? "error"
+                                        : "success"
+                                }
```

Comment:
instead of so many "if-else" you can define an object with keys = status and values = CONSTANTS.something and after that just access the object with "obj[availability]"